tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py27,py36}-django{18,111},py36-django20
+envlist = {py27,py36}-django{18,111},{py35,py36}-django{20,21,22}
```

Comment:
You can drop the Django 1.8 entry from this.  And we should probably switch the Django 1.11 testing from Python 3.6 to 3.5; by the time we upgrade to Python 3.6, we should already be on Django 2.2.